source/documentation/virtual_machine_management_guide/topics/Affinity_Labels.adoc

```diff
@@ -2,11 +2,11 @@
 [id="About_Affinity_Labels"]
 = About Affinity Labels
 
-You can create and modify affinity labels in the Administration Portal.
+You can create and modify *Affinity Labels* in the Administration Portal.
 
-Affinity labels are used to set hard (Enforced) positive affinity between virtual machines and hosts. See the xref:sect-Affinity_Groups[Affinity Groups] section for more information about affinity hardness and polarity.
+*Affinity Labels* are used together with *Affinity Groups* to set hard (Enforced) positive affinity between virtual machines and hosts. See the xref:sect-Affinity_Groups[Affinity Groups] section for more information about affinity hardness and polarity.
```

Comment:
"*Affinity Labels* are used together with *Affinity Groups* to set hard (Enforced) positive affinity between virtual machines and hosts."
Actually I think it must be 
"*Affinity Labels* are used together with *Affinity Groups* to set any kind of affinity between virtual machines and hosts (hard/soft, positive/negative)."

Reply:
thank you!